acquisition/PASSIVE.md

```diff
@@ -35,7 +35,7 @@ Once everything is installed you can start the acquisition server with the
 following commands.
 
     workon tomviz
-    python -m tomviz.acquisition.cli -a tomviz.acquisition.vendors.passive.PassiveWatchSource
+    tomviz.acquisition -a tomviz.acquisition.vendors.passive.PassiveWatchSource
```

Comment:
Should be `tomviz-acquisition`, right?

Reply:
Yes, good catch, thought I had copy-pasted. Thanks.